mne_bids/write.py

```diff
@@ -386,16 +438,31 @@ def _coordsystem_json(raw, unit, orient, manufacturer, fname,
             coords['coil%d' % ident] = hpi[ident]['r'].tolist()
 
     coord_frame = set([dig[ii]['coord_frame'] for ii in range(len(dig))])
-    if len(coord_frame) > 1:
-        err = 'All HPI and Fiducials must be in the same coordinate frame.'
-        raise ValueError(err)
+    if len(coord_frame) > 1:  # noqa E501
+        raise ValueError('All HPI, electrodes, and fiducials must be in the '
+                         'same coordinate frame. Found: "{}"'
+                         .format(coord_frame))
 
-    fid_json = {'MEGCoordinateSystem': manufacturer,
-                'MEGCoordinateUnits': unit,  # XXX validate this
-                'HeadCoilCoordinates': coords,
-                'HeadCoilCoordinateSystem': orient,
-                'HeadCoilCoordinateUnits': unit  # XXX validate this
-                }
+    if kind == 'meg':
+        hpi = {d['ident']: d for d in dig if d['kind'] == FIFF.FIFFV_POINT_HPI}
+        if hpi:
+            for ident in hpi.keys():
+                coords['coil%d' % ident] = hpi[ident]['r'].tolist()
+
+        fid_json = {'MEGCoordinateSystem': coordsystem_name,
+                    'MEGCoordinateUnits': unit,  # XXX validate this
+                    'HeadCoilCoordinates': coords,
+                    'HeadCoilCoordinateSystem': orient,
+                    'HeadCoilCoordinateUnits': unit  # XXX validate this
+                    }
+    elif kind == "ieeg":
+        fid_json = {
+            'iEEGCoordinateSystem': coordsystem_name,  # MRI, Pixels, or ACPC
```

Comment:
what is ACPC?

Reply:
It stands for "anterior commissure and posterior commissure" alignment of the T1 MRI image. It's sort of mentioned here: https://github.com/bids-standard/bids-specification/blob/master/src/04-modality-specific-files/04-intracranial-electroencephalography.md